Frontend/ui-driver/src/MerchantConfigs/Utils/Utils.purs

```diff
@@ -0,0 +1,31 @@
+module  MerchantConfigs.Utils where
+
+import Common.Types.App (LazyCheck)
+import Data.Maybe (Maybe(..))
+import Helpers.Utils (getMerchant, Merchant(..))
+import Screens.Types (Language)
+
+foreign import getString' :: String -> String
+
+foreign import getValueFromMerchant :: String -> String
+
+getLanguage :: LazyCheck -> Array Language
+getLanguage _ = do
```

Comment:
_ is not always effective in case of lazy evaluation may result in no language change